libraries/botbuilder-dialog-prompts/src/email.ts

```diff
@@ -0,0 +1,36 @@
+import * as recognizers  from "@microsoft/recognizers-text-sequence";
+import { Activity, InputHints, TurnContext } from "botbuilder-core";
+import { Prompt, PromptOptions, PromptRecognizerResult, PromptValidator } from "botbuilder-dialogs";
+
+/**
+ * @module botbuildercommunity/dialog-prompts
+ */
+
+export class EmailPrompt extends Prompt<string> {
+    public defaultLocale: string | undefined;
+    constructor(dialogId: string, validator?: PromptValidator<string>, defaultLocale?: string) {
+        super(dialogId, validator);
+    }
```

Comment:
Would it be an idea to add whitespace in between every new function? In my opinion this would increase the readability.

Maybe it would be nice to have an ESLint (or similar tool) configuration which ensures that all packages follow the same code standards.

Reply:
Does TSLint do the same thing, or is it less robust?

Reply:
TSLint would have my preference also! Should I have a look if I can add TSLint to the whole project? I am not sure if every project in `botbuilder-community` should have the same code style. What is your opinion on this?

The preferred config would be the same as [botbuilder-js](https://github.com/Microsoft/botbuilder-js/blob/master/tslint.json) for me, however that is something that should be decided eventually. Let's move that to a seperate PR/issue.

Reply:
I'm fine with TSLint, and I'm fine with using the botbuilder-js configuration for it. In fact, I think that's preferred. I know @garypretty wants to try to keep things as aligned with the Bot Framework team as possible.